### PR TITLE
Milestone 3 Actions

### PR DESCRIPTION
**Pull request by FrancoRosa**
Hello TSEs,
In this milestone I did the following tasks:

- Implement the handlers for input and select on BooksForm.
- Implement a submit handler to save the book on Redux store
- Add a new column to the BooksList component with the remove button.
- Pass the remove handler as props and use and action to remove the book from redux.

Thank you for your kind review and feedback

### CODE REVIEW

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @FrancoRosa  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/containers/BooksForm.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    this.setState({
+      input: event.target.value,
+    });
+  }
+
+  handleSelect(event) {
+    this.setState({
+      select: event.target.value,
+    });
+  }
```
Comment (bellom) [inline src/containers/BooksForm.js]: According to the instruction on pathwright;

we need to make use of `handleChange` method to handle both `input changes` and `select category`.

image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896828/

i.e we could have it this way;

```
handleChange(event) {
  const name = event.target.name;

  this.setState({
    title: event.target.value
    [name]: event.target.value
  })
}
```

**Review by bellom — CHANGES_REQUESTED** [review]

**Comment by FrancoRosa** [comment]
Thank you @bellom for your feedback
The requested improvements were made.

**Review by Torres-ssf — APPROVED** [review]
### Milestone Approved :+1: :hand: 
Hi @FrancoRosa :hand: 
Great job on this milestone and implementing the requested changes.
You may merge the PR.

Happy Coding :computer: 
**Sergio Torres**

**Pull request merged**